pymars/drg.py

```diff
@@ -134,28 +140,37 @@ def trim_drg(total_edge_data, solution_object, threshold_value, keeper_list, don
 
     return exclusion_list
 
-def run_drg(solution_object, conditions, error_limit, target_species, retained_species):
-    """Main function for running DRG reduction.
+
+def run_drg(solution_object, conditions_file, error_limit, target_species,
+            retained_species, model_file, final_error):
+    
```

Comment:
I believe the docstring should immediately follow the function definition, with no blank space in between.

Reply:
Fixed